scaleapi/api.py

```diff
@@ -109,9 +109,28 @@ def _api_request(
         json = None
         if res.status_code == 200:
             json = res.json()
+        elif res.status_code == 409:
```

Comment:
I think our API also returns `409 (Conflict)` during batch and project creation. 
Probably super rarer to have a `500` during batch or project creation, but should we add further governance here to ensure that behavior applies only to task creation?

Checking the `task/` included in `endpoint` in a POST request and checking if `unique_id` is available in the body can be some ways to approach that.

Reply:
good idea, will do that